WordPress/Classes/ViewRelated/Site Creation/SiteSegments/SiteSegmentsWizardContent.swift

```diff
@@ -155,27 +157,56 @@ final class SiteSegmentsWizardContent: UIViewController {
     }
 
     private func handleError(_ error: SiteSegmentsError) {
-        setupEmptyTableProvider()
+        setupErrorView()
     }
 
-    private func setupEmptyTableProvider() {
-        let message: InlineErrorMessage
-        if isNetworkActive {
-            message = InlineErrorMessages.serverError
+    private func setupErrorView() {
+        let errorType: ErrorStateViewType
+        if self.isNetworkActive == false {
+            errorType = .networkUnreachable
         } else {
-            message = InlineErrorMessages.noConnection
+            errorType = .siteLoading
```

Comment:
While your approach works here, the `.siteLoading` type includes both the dismissal `X` button and the `Contact Support` label, and is intended for the final step in the site creation flow.

Consider using the `.general` case instead. This will help us if the title & subtitle should diverge in the future.